schema-registry/src/main/java/io/streamnative/pulsar/handlers/kop/schemaregistry/resources/SchemaResource.java

```diff
@@ -92,7 +100,30 @@ protected CompletableFuture<GetSchemaResponse> processRequest(Void payload, List
                 if (s == null) {
                     return null;
                 }
-                return new GetSchemaResponse(s.getSchemaDefinition());
+                return new GetSchemaResponse(s.getSchemaDefinition(), s.getType());
```

Comment:
> In which case could s.getType() return a non-null value?

If the schema type is not AVRO, the type will be returned.

Set `s.getType()` here just to avoid code like 

```
if ("AVRO".equals(s.getType()) {
        return new GetSchemaResponse(s.getSchemaDefinition());
} else {
        return new GetSchemaResponse(s.getSchemaDefinition(), s.getType();
}
```

https://github.com/streamnative/kop/pull/1841/files#diff-f89b0d1bfe575f4b8055f8ea95badf2d179ab1374bfdd098d26c89238ff8c26bR59 will handle the final response data. 

It's fine for me to change it back until we have other schema types. @BewareMyPower WDYT?

The good thing is if we have it here, we don't need to change the code here for other schema types in the future. Otherwise, we might miss the schema type field for non-avro schema.

Reply:
It makes sense to me. Just keep it here.